admin/src/components/NavBar.spec.js

```diff
@@ -68,23 +68,20 @@ describe('NavBar', () => {
   })
 
   describe('wallet', () => {
-    const windowLocationMock = jest.fn()
     const windowLocation = window.location
     beforeEach(async () => {
       delete window.location
-      window.location = {
-        assign: windowLocationMock,
-      }
-      await wrapper.findAll('.nav-item').at(5).find('a').trigger('click')
+      window.location = ''
+      await wrapper.findAll('.nav-item').at(4).find('a').trigger('click')
     })
 
     afterEach(() => {
       delete window.location
       window.location = windowLocation
     })
 
-    it.skip('changes window location to wallet', () => {
-      expect(windowLocationMock()).toBe('valid-token')
+    it('changes window location to wallet', () => {
+      expect(window.location).toBe('http://localhost/authenticate?token=valid-token')
```

Comment:
Is this always an URL with localhost? What happens on other instances like stage1/2/3 or production?